BoardingPass/BoardingNavigationController.swift

```diff
@@ -79,9 +149,9 @@ public class BoardingNavigationController: UINavigationController {
      - parameter animated: Specify true to animate the transition or false
      if you do not want the transition to be animated.
      */
-    public func pushToNextViewController(animated animated: Bool) {
-        if let pushableViewController = topViewController.flatMap(boardingInfoAfter) {
-            pushViewController(pushableViewController, animated: animated)
+    public func pushToNextViewController(animated isAnimated: Bool) {
+        if let pushableViewController = topViewController.flatMap({ boardingInfo(afterController: $0) }) {
```

Comment:
#? I think `.flatMap( boardingInfo(afterController: ))` will work without having to wrap it in a closure